cypress/integration/visitorCanSeeMainPage.feature.js

```diff
@@ -0,0 +1,37 @@
+describe('Visitor can see main page', () => {
+  describe('success', () => {
+    beforeEach(() => {
+      cy.intercept('GET', 'https://fake-newzzzz.herokuapp.com/api/articles', {
+        fixture: 'articles.json',
+      });
+      cy.visit('/');
+    });
+    it('displays main page layout', () => {
+      cy.get('[data-cy=navbar]').should('exist');
+      cy.get('[data-cy=header]').should('exist');
+      cy.get('[data-cy=breaking-news]').should('exist');
+      cy.get('[data-cy=breaking-img]').should('exist');
+      cy.get('[data-cy=breaking-img]').should('exist');
+      cy.get('[data-cy=breaking-title]').should('contain', 'Something');
+      cy.get('[data-cy=breaking-teaser]').should('contain', 'Something more');
```

Comment:
I would update the title to include "...with a breaking content". But I also question whether this is actually in scope? I see no categories in your fixture so how do you determine that an article should be featured as "breaking"?

Reply:
All the tests been refactored now and removed unnecessary tests and rearranged them.